ReCaptchaFrontendUi/view/frontend/web/js/reCaptcha.js

```diff
@@ -10,9 +10,10 @@ define(
         'uiComponent',
         'jquery',
         'ko',
-        'Magento_ReCaptchaFrontendUi/js/registry'
+        'Magento_ReCaptchaFrontendUi/js/registry',
+        'Magento_ReCaptchaFrontendUi/js/reCaptchaScriptLoader',
```

Comment:
there is mistake, not sure how it could work
```suggestion
        'Magento_ReCaptchaFrontendUi/js/reCaptchaScriptLoader'
```